features/b.yml.dist

```diff
@@ -0,0 +1,17 @@
+# Generated from: b.yml
```

Comment:
@petele can you remove the orphaned features/draft/html-elements/b.yml.dist file?

Reply:
done!